src/test/crimson/test_messenger.cc

```diff
@@ -54,19 +54,18 @@ static entity_addr_t get_server_addr() {
   return saddr;
 }
 
+template <typename T>
+static std::list<seastar::lw_shared_ptr<seastar::sharded<T>>> sharded_objects;
+
 template <typename T, typename... Args>
 seastar::future<T*> create_sharded(Args... args) {
-  // we should only construct/stop shards on #0
```

Comment:
Let's keep the comment, otherwise it would be hard to understand what is going on

Reply:
Yeah, seems true for `seastar::sharded` when I was implementing this helper. Supposing that this fact was not changed.